helix-view/src/theme.rs

```diff
@@ -157,9 +165,34 @@ fn parse_color(value: Value) -> Option<Color> {
         if let Some((red, green, blue)) = hex_string_to_rgb(&s) {
             Some(Color::Rgb(red, green, blue))
         } else {
+            warn!("malformed hexcode: {}", s);
             None
         }
     } else {
+        warn!("unrecognized value: {}", value);
```

Comment:
Lacking context in error message. Like "unrecognized value: {}", so random. The next step is like point to the line or something for more context. Not sure if we want to do this now or defer it, if you can do it maybe you can try or else add a TODO maybe?

Reply:
I think it's out of scope for the PR. I added a TODO, and a little more context to the warnings. I'll come back to it when we have a facility for alerting the user.